Src/CSharpier.Cli/Options/ConfigurationFileOptions.cs

```diff
@@ -16,6 +16,8 @@ internal class ConfigurationFileOptions
     [JsonConverter(typeof(CaseInsensitiveEnumConverter<EndOfLine>))]
     public EndOfLine EndOfLine { get; init; }
 
+    public bool UsePrettierStyleTrailingCommas { get; init; }
```

Comment:
I'd get rid of the option and just always format this way. Prettier needed it as an option because fo the different flavors of ES. If some day c# adds new trailing comma support that isn't in old versions then CSharpier may need an option for "CSharpVersion".

Reply:
I absolutely agree with you about the decision. Removed the option.